security/cmd/istio_ca/main.go

```diff
@@ -262,6 +263,16 @@ func init() {
 	cmd.InitializeFlags(rootCmd)
 }
 
+func initEnvVars() {
+	selfNamespace := env.RegisterStringVar("POD_NAMESPACE", "",
+		"Namespace this instance of Citadel is running in").Get()
+	enableNamespacesByDefault := env.RegisterBoolVar("CITADEL_ENABLE_NAMESPACES_BY_DEFAULT", true,
```

Comment:
Same comment about the env variables - it should match the option in values.yaml ( with CITADEL_ prefix as is the pattern )